src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
you shouldn't do this for `*` because it affects performance

Reply:
changed